Index.html

```diff
@@ -53,30 +53,29 @@ <h3>Games</h3>
           <li>Darkest Dungeon</li>
         </ol>
         <h3>Haiku</h3>
-        <p>Majestic Sages</p>
-        <p>Keepers of Ancient Wisdom</p>
-        <p>It's snowing on Mt. Fuji.</p>
-        <cite>by <span class="author">Unknown</span></cite>
+        <p>
+          Majestic Sages<br>
+          Keepers of Ancient Wisdom<br>
+          It's snowing on Mt. Fuji.<br>
+          <cite>by <span class="author">Unknown</span></cite>
+        </p>
       </main>
       <footer>
         <hr />
         <blockquote>
           Man, I hate druids. Who designed this class? Literally burst healing
           over time and manditory raid and dungeon healer for 4 years.
+          <cite>&mdash; <span class="author">Me(The Druid Hater)</span></cite>
         </blockquote>
-        <cite
-          ><em>&macr; <span class="author">Me(The Druid Hater)</span></em></cite
-        >
         <p>
-          &mdash; 2020
+          &copy; 2020
           <a
             href="mailto:Soltaege@gmail.com?subject=Message%20from%20CSD120%20Website"
-            >Erika Taege</a
-          >
+            >Erika Taege</a>
```

Comment:
Odd formatting here and in other places.  Please be sure to run a code formater such as the  VS Code extension "Prettier" on your code before submitting